TACC/Core-Styles
# fix(demo)!: font path change in dist

## Overview

Change the font paths in `dist/`.

> **⚠️ Warning**
> Clients replacing `../../fonts`—when loading from __`dist/`__—must now replace `../fonts`.

> **✅ Notice**
> Clients replacing `../../fonts`—when loading from __`src/`__—may still do so safely.

## Related

Recent changes also about fonts:
-  #72
- #71
- #70

## Changes

- add `postcss-replace` command to swap `../../` with `../`
- build `dist/`

## Testing

1. `npm run build && npx serve demo`
    <sup>(`npx` may temporarily download an HTTP server package)</sup>
2. open http://localhost:3000/components/detail/s-affixed-input-wrapper
    <sup>(page should load as normal)</sup>
3. inspect a `<dt>` element in the pattern e.g. "Prefix"
    <sup>(using browser's Developer Tools)</sup>
4. verify `font-family` is `Arial`
    <sup>(in "Computed" panel, next to "Styles" panel)</sup>
5. (optional) see Roboto font is loaded
    <sup>(in "Network" panel, from Google)</sup>
6. **disable `font-family` CSS rule**
    <sup>(in "Styles" panel, uncheck the rule)</sup>
7. **verify `font-family` is `Benton Sans`**
    <sup>(in "Computed" panel, next to "Styles" panel)</sup>
8. **verify Benton Sans font is loaded**
    <sup>(in "Network" panel, from localhost)</sup>

## UI

| | styles | computed | network |
| - | - | - | - |
| before | ![before (Styles)](https://user-images.githubusercontent.com/62723358/203601743-fc5982ec-40a9-4d2c-b10a-4099f7933aa2.png) | ![before (Computed)](https://user-images.githubusercontent.com/62723358/203601737-c72b8137-5e4b-4cc4-9ea6-e0797d531557.png) | ![before (Network)](https://user-images.githubusercontent.com/62723358/203602577-d8c38de2-1463-4253-b612-73ebdbfbdcf2.png) |
| after | ![after (Styles)](https://user-images.githubusercontent.com/62723358/203601729-795b8e09-4cf8-41ea-9034-73091b92ab3b.png) | ![after (Computed)](https://user-images.githubusercontent.com/62723358/203601724-66ecca91-6b22-4d77-abd9-8ae4e856e2dc.png) | ![after (Network)](https://user-images.githubusercontent.com/62723358/203602611-8e766b85-556e-4ce5-b88d-73a8b432d443.png) |
